### PR TITLE
Improve documentation

### PR DESCRIPTION
**Pull request by hlindberg**
This PR makes some small changes to the documentation for things that tripped me up when I started exploring dgraph's features.
Hope you also find them useful. See commit comments per commit for rationale for the changes.

---
This change is [Reviewable](https://reviewable.io/reviews/dgraph-io/dgraph/4223)

### CODE REVIEW

File: wiki/content/query-language/index.md [hunk 1]
```diff
@@ -2374,6 +2374,50 @@ The reverse edge of `anEdge` is `~anEdge`.
...
+```
+
+Makes it possible to query Persons and their cars by using:
+```
+q(func type(Person)) {
+  name
+  ~owner { name }
+}
+```
+And to get a different key than `~owner` in the result the query can be written with the wanted label (`cars`
```
[the remark this answers is not in the document]
Comment (hlindberg) [reply wiki/content/query-language/index.md]: ok, thanks, will fix.

File: wiki/content/query-language/index.md [hunk 2]
```diff
@@ -2374,6 +2374,50 @@ The reverse edge of `anEdge` is `~anEdge`.
...
+in this case):
+
+```
+q(func type(Person)) {
+  name
+  cars: ~owner { name }
+}
+```
+
+This also works if there can be multiple "owners" of a car
```
[the remark this answers is not in the document]
Comment (hlindberg) [reply wiki/content/query-language/index.md]: will add a colon

**Comment by MichelDiz** [comment]
Is this PR still valid? There are fix and conflicts to be solved.

**Comment by hlindberg** [comment]
Hi, sorry - I forgot about this. Will rebase and see if still relevant.

**Comment by bucanero** [comment]
> Hi, sorry - I forgot about this. Will rebase and see if still relevant.

Hello @hlindberg, any update regarding this PR?

**Comment by hlindberg** [comment]
I made a new PR with the rebased changes. See #7185

**Pull request closed without merging**